base/domain-zul.yml

```diff
@@ -0,0 +1,376 @@
+    auto_fill: false
+  location_confirm:
+    type: unfeaturized
+    auto_fill: false
+  location_coords:
+    type: unfeaturized
+    auto_fill: false
+  city_location_coords:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_obesity:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_diabetes:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_hypertension:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_cardio:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_fever:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_cough:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_sore_throat:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_difficulty_breathing:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_taste_smell:
+    type: unfeaturized
+    auto_fill: false
+  exposure:
+    type: unfeaturized
+    auto_fill: false
+  tracing:
+    type: unfeaturized
+    auto_fill: false
+  requested_slot:
+    type: unfeaturized
+
+forms:
+  - healthcheck_form
+  - healthcheck_terms_form
+  - healthcheck_profile_form
+
+responses:
+  utter_welcome:
+      - text: |
+          *HealthCheck*  iyithuluzi lakho lokuhlola ubungozi njalo ngeviki. Sicela usisize ngokuphendula imibuzo embalwa ngawe nangempilo yakho. Uzothola izeluleko zokuthi yiziphi izinyathelo okufanele uzithathe ngokususelwa kuzinkombandlela zikazwelonke nemininingwane oyifakayo izosisiza ekwakheni imodeli nasekuhleleni impendulo yethu kazwelonke ku-COVID-19. Siyabonga ngokuza phambili nokuba neqhaza empilweni yazo zonke izakhamizi nokuvimbela ukusabalala kwegciwane 🦠
+
+  utter_welcome_back:
+      - text: |
+          Welcome back to *HealthCheck*, your weekly risk assessment tool. Thank you for coming forward and for contributing to the health of all citizens AND stopping the spread of the virus 🦠
+
+  utter_ask_terms:
+      - text: |
+          Phendula ngelinye lamagama eligqamile ukuze uqhubeke
+
+          MUKELA✅ -_ukwamukela imigomo nemibandela_
+          OKWENGEZIWE 📄 -_ukufunda imigomo nemibandela_
+
+          (noma Phendula ngo-UHLU 📌 _ukubuyela kuhlu lokuqala_)
+
+  utter_more_terms:
+      - text: |
+          Uqinisekisa ukuthi igunya elakho ekunakekelweni nokwelashwa kwakho. Le nsizakalo inikezela ngolwazi kuphela.
+
+          Lensizakalo ayiyona indawo yokwelulekwa ngokwezempilo/ukuzixilonga/ukwelashwa. Thola izeluleko zomhlinzeki wezempilo ofanele mayelana nesimo sakho sezokwelashwa/ ukunakekelwa, ikakhulukazi uma uba nezimpawu ezinzima.
+
+          Uqinisekisa ukuthi akufanele unganaki/ ubambezele ukufuna izeluleko zezokwelashwa mayelana nokwelashwa/ukunakekelwa ngenxa yale nsiza. Thembela kulwazi ngokuqapha.
+
+          Phendula ngelinye lamagama *ELIGQAMILE* ukuze uqhubeke
+          *MUKELA* ✅ ukwamukela imigomo nemibandela
+          (noma Phendula 📌 *UHLU* ukubuyela kuhlu lokuqala)
```

Comment:
This additional section at the end, which is a repeat of `utter_ask_terms`, needs to be removed. `utter_ask_terms` is repeated after this message